src/ch/ehi/interlis/domainsandconstants/linetypes/LineType.java

```diff
@@ -453,5 +453,17 @@ public void _unlinkControlpoints(DomainDef controlpoints1)
 
                 // -end- 358A65070007 detail_end "LineType"
 
+                private boolean isMulti;
+
+                public boolean isMulti() {
+                    return isMulti;
+                }
+
+                public void setMulti(boolean isMulti) {
```

Comment:
richtig wäre:

    if(isMulti != value1){
      isMulti= value1;
      ch.ehi.uml1_4.changepropagation.MetaModel.getInstance().notifyChange(new ch.ehi.uml1_4.changepropagation.MetaModelChange(this,"setMulti"));
    }